js/main.js

```diff
@@ -0,0 +1,27 @@
+
+//Функция, возвращающая случайное целое число из переданного диапазона включительно
+function returnsRandomNumberFromInterval(min, max) {
+
+  //Проверка числа на >= 0 и корректность диапазона
+  if (min >= 0 && max >= 0 && min < max) {
+    /*округление и возвращение случайного числа
+      формула с текста "Math.random" отсюда -
+      https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+      */
```

Comment:
```suggestion
    // округление и возвращение случайного числа
    // формула с текста "Math.random" отсюда -
    //  https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
```

Многострочный комментарий рекомендуется использовать для описания документации)